src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -485,7 +485,6 @@ void convertConstructorToClass(Node n, JSDocInfo jsDoc) {
 
     typeNode.setJSDocInfo(n.getJSDocInfo());
     nodeComments.replaceWithComment(n, typeNode);
-    compiler.reportCodeChange();
```

Comment:
why is it ok to sometimes 'reportCodeChange' and sometimes just remove it? I might be lacking some basic understanding of how closure's AST mutation works.

Reply:
To be honest I'm not sure. It might be that there's another change report in the same change scope making this one unnecessary. It also might be that there really *should* be a report here, but since this is a standalone tool rather than an optimization pass in the compiler, it's not that critical anyway.